src/platforms/php/guides/symfony/index.mdx

```diff
@@ -21,28 +21,25 @@ composer require sentry/sentry-symfony
 
 Add your DSN to `app/config/config.yml`:
 
-```yaml {filename:app/config/config.yml}
+```yaml {filename:config/config.yml}
 sentry:
-  dsn: "___PUBLIC_DSN___"
+  dsn: "%env(SENTRY_DSN)%"
```

Comment:
We use the `___PUBLIC_DSN___ ` "magic keyword" in docs to automatically fill in a DSN for logged in Sentry.io users.

```suggestion
  dsn: "___PUBLIC_DSN___"
```

Reply:
The `___PUBLIC_DSN___` keyword is still present, this PR moves it to the .env file that's used to hold secrets and other environment specific configuration. `SENTRY_DSN` is defined there.
`%env(SENTRY_DSN)` is the Symfony way of saying "insert the env variable `SENTRY_DSN` here"

Reply:
Thanks for chiming in @fridtjof. If that's the preferred way in Symfony, then we should align the content here and `src/wizard/php/symfony.md`. In the wizard there's a note:

> If you're using the Symfony Flex plugin, you'll find this file already created for you; it will suggest using an environment variable to inject the DSN value securely.

I wonder if the env var approach also works and the `.env` file is picked up automatically if not using the Flex plugin?

Reply:
>I wonder if the env var approach also works and the `.env` file is picked up automatically if not using the Flex plugin?

Flex is just a way to automate installation/uninstallation of packages, the functionality of loading the `.env` files is provided by the framework itself